openassessment/templates/openassessmentblock/grade/oa_grade_waiting.html

```diff
@@ -20,8 +20,16 @@ <h2 class="step__title">
                         <p>{% trans "Your response is still undergoing peer assessment. After your peers have assessed your response, you'll see their comments and receive your final grade." %}</p>
                     {% elif waiting == 'example-based' %}
                         <p>{% trans "Your response is still undergoing example-based assessment. After your response has been assessed, you'll see the comments and receive your final grade." %}</p>
+                    {% elif waiting == 'staff' %}
+                        <p>{% trans "Your response is still undergoing staff assessment. After your response has been assessed, you'll see their comments and receive your final grade." %}</p>
+                    {% elif waiting == 'peer,example-based' %}
+                        <p>{% trans "Your response is still undergoing peer assessment and example-based assessment. After your your peers have assessed your response and example-based assessment is complete, you'll see your peers' comments and receive your final grade." %}</p>
+                    {% elif waiting == 'peer,staff' %}
+                        <p>{% trans "Your response is still undergoing peer assessment and staff assessment. After your peers and instructors have assessed your response, you'll see your peers' and instructors' comments and receive your final grade." %}</p>
```

Comment:
Remove one instance of "assessment" in the 1st sentence ("Your response is still undergoing peer and staff assessment.")
Replace 2nd sentence with: "After your peers and course staff have assessed your response, you will see their comments and receive your final grade."